src/frontend/komponenter/Behandling/Blankett/Blankett.tsx

```diff
@@ -41,16 +40,14 @@ const Blankett: React.FC<Props> = ({ behandlingId }) => {
     return (
         <>
             <DataViewer response={{ behandling }}>
-                {({ behandling }) => (
-                    <>
-                        <StyledBlankett>
-                            <PdfVisning pdfFilInnhold={blankettRessurs}></PdfVisning>
-                        </StyledBlankett>
-                        {erBehandlingRedigerbar(behandling) && harHentetBlankett() && (
-                            <SendTilBeslutterFooter behandlingId={behandlingId} />
-                        )}
-                    </>
-                )}
+                <>
```

Comment:
Usikker, men trenger vi DataViewer her nå? 

Noe jeg tror:
Ja - fordi vi må "vente" på å hente behandlingen for å få satt behandlingErRedigerbar riktig? Hvis ikke vises knappen først, før den så forsvinner?
Nei - fordi knappen uansett ikke vises, så den kan fjernes.


Jeg vet ikke - må testes/evt. om du har en bedre forklaring?

Reply:
True, fjerner